### DISCUSSION TITLE
Reauthentication

### DISCUSSION THREAD

**Discussion by inderpalpatheja** [strongswan/strongswan#2232] [Q&A]
Hi Team,

Need one help. In my usecase there is one VPN server which is a responder and handling VPN clients.
Now is there a way which i can trigger force reauthentication from responder after the certificates on responder side are renewed/changed?

I have been trying below command but it is not triggering the re-authentication
swanctl --rekey --ike home1 --reauth

Please help or suggest some ideas

**Comment by tobiasbrunner** [comment]
You could just terminate their IKE_SAs (if you know which ones and if the clients will reconnect automatically).

> I have been trying below command but it is not triggering the re-authentication

For usual roadwarrior scenarios this won't work because only clients can actively reauthenticate in such asymmetric setups.

Immediately requesting a reauthentication from clients on established IKE_SAs is currently only possible by setting a low lifetime via [RADIUS DAE](https://docs.strongswan.org/docs/5.9/plugins/eap-radius.html#_session_timeout_and_dynamic_authorization_extension) and only if the clients support RFC 4478 (which defines the `AUTH_LIFETIME` notify that is also used by the server to [request a reauthentication](https://docs.strongswan.org/docs/5.9/config/rekeying.html#_ikev2_responder_behavior) after a configured time).

**Comment by inderpalpatheja** [comment]
Thanks for your quick response.
How will clients connect automatically ? Do we need to configure anything
in client configuration to initiate the connection again?

regards
Inder

**Reply by tobiasbrunner** [reply]
If you use trap policies, the connection can be triggered by traffic. Then there is also `close_action=restart`. Or you could handle it actively via vici.
